staging/src/k8s.io/component-base/config/v1beta1/types.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v1alpha1
+package v1beta1
 
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
the ResourceLock field needs to document
* the allowed values (`leases`, `endpointsleases`, `endpoints`, `configmapsleases`, and `configmaps`)
* the recommended value if backwards compatibility with an older config is not required (`leases`)
* the recommended value if backwards compatibility with an older config using `endpoints` or `configmaps` is required (`endpointsleases` or `configmapsleases`, respectively)